cmd/agent/main.go

```diff
@@ -222,54 +222,73 @@ func main() {
 	log := zap.S()
 	defer log.Sync()
 
-	transportConf := transport.PlatformGRPCConf{
-		UUID:            global.AgentHostname,
-		APIKey:          global.AgentConf.Platform.APIKey,
-		TransmitTimeout: global.AgentConf.Platform.TransportTimeout,
-		URL:             global.AgentConf.Platform.Addr,
-	}
+	if global.AgentConf.Platform.Enabled() {
```

Comment:
Do you think we could move code inside this `if` block somewhere else?

Reply:
Addressed in 9400c91